build.gradle

```diff
@@ -122,7 +122,7 @@ ext {
             "search"  : "./src/search/crafter-search-server/target/crafter-search.war",
             "profile" : "./src/profile/server/target/crafter-profile.war",
             "engine"  : "./src/engine/target/ROOT.war",
-            "deployer": "./src/deployer/target/crafter-deployer.jar",
+            "deployer": "./src/deployer/target/crafter-deployer-exec.jar",
```

Comment:
Why add -exec.jar? What is this needed for?

Reply:
While doing tests I found that the spring-boot executable jars can't be used as a maven dependency in other projects, so we need to generate both of them and now `crafter-deployer.jar` is the regular one.

Reply:
We can't rename the executable one, this would break all existing installations. If we need a new jar with only classes rename that one to `crafter-deployer-classes.jar`.

Reply:
There is already one with `-sources` but that one doesn't work as dependency either, spring-boot-maven-plugin repackages the regular jar so we only have control over the executable

Reply:
I also added a rename so the result of the deploy is the same as before

Reply:
Ok, seems fair